openmmtools/forcefactories.py

```diff
@@ -74,6 +75,58 @@ def replace_reaction_field(reference_system, switch_width=1.0*unit.angstrom,
     return system
 
 
+# =============================================================================
+# RESTRAIN ATOMS
+# =============================================================================
+
+def restrain_atoms(thermodynamic_state, sampler_state, mdtraj_topology,
+                   atoms_dsl, sigma=3.0*unit.angstroms):
+    """Apply a soft harmonic restraint to the given atoms.
+
+    This modifies the ``ThermodynamicState`` object.
+
+    Parameters
+    ----------
+    thermodynamic_state : openmmtools.states.ThermodynamicState
+        The thermodynamic state with the system. This will be modified.
+    sampler_state : openmmtools.states.SamplerState
+        The sampler state with the positions.
+    mdtraj_topology : mdtraj.Topology
+        The topology.
+    atoms_dsl : str
+        The MDTraj DSL string for selecting the atoms to restrain.
+    sigma : simtk.unit.Quantity, optional
+        Controls the strength of the restrain. The smaller, the tighter
+        (units of distance, default is 3.0*angstrom).
+
+    """
+    K = thermodynamic_state.kT / sigma  # Spring constant.
```

Comment:
Uh oh! Yes it should. I guess I didn't notice it in the original script. Fixing it!